README.md

```diff
@@ -1,2 +1,52 @@
 # pwa-prototype-reporting
-A playground for a PWA for reporting only purposes. 
+A playground for a PWA for reporting only purposes. This project started as an experimental prototype to migrate the functionality of Ushahidi survey submissions into a small standalone project that can be deployed anywhere and in the future could be linked to any ushahidi.io deployment chosen by the user.
```

Comment:
Please remove `A playground for a PWA for reporting only purposes.`   :)

Reply:
Removed it.